src/notebook_index.ipynb

```diff
@@ -114,6 +114,8 @@
     "  * [pump](Examples/UnitModels/Operations/pump.ipynb) - Pump unit model with iapws property package\n",
     "  * [heat exchanger 0D](Examples/UnitModels/Operations/heat%20exchanger%200D.ipynb) - Heat Exchanger 0D unit model heating a benzene-toluene mixture using steam\n",
     "  * [skeleton unit](Examples/UnitModels/Operations/skeleton%20unit.ipynb) - Skeleton unit model demonstrating custom variables, ports, and constraints\n",
+    "  * [statejunction](Examples/UnitModels/Operations/statejunction.ipynb) - Statejunction unit model with modular property package\n",
```

Comment:
I would remove the "unit model" from this line - as mentioned earlier a `StateJunction` is not a unit model (it is a convenience originally requested by GDP applications).